lib/phoenix/live_dashboard/live/menu_live.ex

```diff
@@ -22,8 +22,10 @@ defmodule Phoenix.LiveDashboard.MenuLive do
     <nav id="menu-bar">
       <%= maybe_active_live_redirect @socket, @menu, "Home", :home, @node %>
       <%= maybe_enabled_live_redirect @socket, @menu, "OS Data", :os_mon, @node %>
-      <%= maybe_enabled_live_redirect @socket, @menu, "Metrics", :metrics, @node %>
-      <%= maybe_enabled_live_redirect @socket, @menu, "Request Logger", :request_logger, @node %>
+      <%= if @menu.dashboard_running? do %>
```

Comment:
I am not sure if we should fully hide those menu items... but I think it is good enough for now.